README.md

```diff
@@ -0,0 +1,185 @@
+# ami-builder-configs
+Configuration files for building various AMIs using ami-builder
+
+All our AMI configuration source code is kept in [AMI Builder Config](https://github.com/dwp/ami-builder-configs)
+
+# ami-builder
+Building AMIs using a Lambda function
+
+## How to Use This Code
+
+### Deploy the Lambda Code
+Step 1: Grab the latest Packer release [release zip file](https://github.com/dwp/ami-builder/blob/master/.circleci/config.yml)
+This is done via the CircleCI config.yml file, and then published on GitHub.
+Example below:
+```
+mkdir artifacts
+sudo pip install -r requirements.txt -t artifacts
+cp -v ami_builder.py LICENSE packer_template.json.j2 README.md artifacts
+curl -O https://releases.hashicorp.com/packer/1.3.3/packer_1.3.3_linux_amd64.zip
+cd artifacts
+unzip ../packer_1.3.3_linux_amd64.zip
+LATEST_VERSION=$(curl --silent "https://api.github.com/repos/${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME}/releases/latest" | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/')
+VERSION=$(echo $LATEST_VERSION | awk -F. '{$NF = $NF + 1;} 1' | sed 's/ /./g')
+VERSION_NUMBER=$(echo $VERSION | sed 's/^v\(.*\)$/\1/')
+zip -r ami-builder-$VERSION_NUMBER.zip *
+- persist_to_workspace:
+root: artifacts
+paths:
+- ami-builder-*.zip
+publish-github-release:
+docker:
+- image: cibuilds/github:0.10
+steps:
+ - attach_workspace:
+at: ./artifacts
+- run:
+name: "Publish Release on GitHub"
+command: |
+LATEST_VERSION=$(curl --silent "https://api.github.com/repos/${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME}/releases/latest" | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/')
+VERSION=$(echo $LATEST_VERSION | awk -F. '{$NF = $NF + 1;} 1' | sed 's/ /./g')
+echo "ghr -t GITHUB_TOKEN -u ${CIRCLE_PROJECT_USERNAME} -r ${CIRCLE_PROJECT_REPONAME} -c ${CIRCLE_SHA1} -delete ${VERSION} ./artifacts/"
+ghr -t ${GITHUB_TOKEN} -u ${CIRCLE_PROJECT_USERNAME} -r ${CIRCLE_PROJECT_REPONAME} -c ${CIRCLE_SHA1} -delete ${VERSION} ./artifacts/
+
+```
+
+Step 2: Create an AWS Lambda function.
+
+[Packer Lambda Code](https://github.ucds.io/dip/aws-management-infrastructure/blob/master/packer.tf)
+Creating the Lambda function
+
```

Comment:
I would probably note, just for the sake of having done so, that the following code is Terraform examples.